src/renderer/corporate/common/common-helpers.jsx

```diff
@@ -11,13 +13,19 @@ const getEntityIcon = entry => {
 
 const getEntityName = entry => {
 	if (entry.identity.type === 'individual') {
-		return `${entry.identity.lastName}, ${entry.identity.firstName}`;
+		return `${entry.lastName}, ${entry.firstName}`;
 	} else {
 		return `${entry.entityName}`;
 	}
 };
 
-const getEntityEmail = entry => entry.email;
+const getEntityEmail = entry => {
+	const idAttribute = 'http://platform.selfkey.org/schema/attribute/email.json';
```

Comment:
should use constant here

Reply:
isn't entry is a profile here? In that case it should have an 'email' field

Reply:
Common helpers is refactored on edit-members pull request